ingest/src/main/java/com/connexta/ingest/controllers/IngestController.java

```diff
@@ -37,19 +36,20 @@ public IngestController(IngestService ingestService) {
     String mediaType = multipartFile.getContentType();
     String fileName = multipartFile.getOriginalFilename();
     Long fileSize = multipartFile.getSize();
-    log.info("Attempting to ingest {}", fileName);
+    log.info(
+        "Ingest request received acceptVersion={}, fileName={}, fileSize={}, mediaType={}",
+        acceptVersion,
+        fileName,
+        fileSize,
+        mediaType);
+    InputStream inputStream;
     try {
-      ingestService.ingest(fileSize, mediaType, multipartFile.getInputStream(), fileName);
-    } catch (IOException | StoreException | TransformException e) {
-      log.warn(
-          "Unable to complete ingest request with params acceptVersion={}, fileSize={}, mediaType={}, fileName={}",
-          acceptVersion,
-          fileSize,
-          mediaType,
-          fileName,
-          e);
-      return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
+      inputStream = multipartFile.getInputStream();
+    } catch (IOException e) {
+      throw new ValidationException("Could not open attachment");
```

Comment:
❓ Why not piggy back the cause exception? This is a great helper when debugging.